### ISSUE TITLE
Blazor daily tests is broken

### ISSUE THREAD

**Issue by ajaybhargavb** [dotnet/aspnetcore#22556]
Blazor daily test pipeline is failing to build with the following error,
```
##[error].dotnet\sdk\5.0.100-preview.6.20266.3\Microsoft.Common.CurrentVersion.targets(4650,5): error MSB3030: (NETCORE_ENGINEERING_TELEMETRY=Build) Could not copy the file "F:\workspace\_work\1\s\artifacts\bin\InProcessRequestHandler\Win32\Release\aspnetcorev2_inprocess.dll" because it was not found.
```

https://dev.azure.com/dnceng/internal/_build/results?buildId=672214&view=logs&j=2d510120-05fb-5ea7-9815-c1a4f41baded&t=827cbafc-16dd-5965-53f8-5da2b2c53942

**Comment by ajaybhargavb** [comment]
@dotnet/aspnet-build have you seen this error recently? Seems unrelated to Blazor

**Comment by ajaybhargavb** [comment]
FYI Blazor folks, @pranavkm @captainsafia @SteveSandersonMS

**Comment by dougbu** [comment]
This pipeline was apparently broken by my `dotnet msbuild` change. `build.ps1 -projects` only builds the specified projects in `dotnet msbuild` or (if you also specify `-buildNative`) in desktop `msbuild`. Desktop `msbuild` doesn't support `net5.0` on the current build agents.

The fix here would be similar what I did in eng/scripts./GenerateReferenceAssemblies.ps1: Run the root `build.ps1` with just `-buildNative`  before building the projects. Or, I might change the root script to always do a native build unless you specify `-noBuildNative`.

@mkArtakMSFT there's a break in the daily-dev-builds pipeline due to a likely-separate issue with this script. Are we still using that pipeline❔ The question was asked a couple of weeks ago but I'm forgetting the final decision.

**Comment by mkArtakMSFT** [comment]
I think the agreement was to disable the pipeline given it serves no good reason at the moment. Please go ahead and disable it, @dougbu.

**Comment by dougbu** [comment]
Saw your email but &hellip; tomorrow 😺

**Comment by dougbu** [comment]
#22611 is ready for review and fixes this issue. Just trying to get past some flakiness, a lot of which points at the Razor SDK build.

#22639 gets rid of the aspnetcore-daily-dev-builds schedule. Hitting some of the same flakiness.

**Comment by dougbu** [comment]
Main issue fixed in ee80cd5ccdbe
